--- origsrc/libmodplug-0.8.7/src/Makefile.am	2009-04-22 18:51:16.000000000 -0500
+++ src/libmodplug-0.8.7/src/Makefile.am	2009-07-13 16:03:14.501247000 -0500
@@ -1,9 +1,7 @@
 AM_CPPFLAGS = -I$(top_srcdir)/src/libmodplug
 
-if MINGW32
-  AM_LDFLAGS = -no-undefined
-endif
 lib_LTLIBRARIES = libmodplug.la
+libmodplug_la_LDFLAGS = -no-undefined
 libmodplug_la_LIBADD = -lm
 libmodplug_la_SOURCES = tables.h         \
                         sndmix.cpp         \
