--- origsrc/libmodplug-0.8.4/src/Makefile.am	2006-11-01 21:48:13.000000000 -0600
+++ src/libmodplug-0.8.4/src/Makefile.am	2007-11-09 02:39:20.562500000 -0600
@@ -1,5 +1,6 @@
-INCLUDES = -Ilibmodplug
+INCLUDES = -I$(srcdir)/libmodplug
 lib_LTLIBRARIES = libmodplug.la
+libmodplug_la_LDFLAGS = -no-undefined
 libmodplug_la_LIBADD = -lm
 libmodplug_la_SOURCES = tables.cpp         \
                         sndmix.cpp         \
