Added in 0.8.8.5; drop if binary compatibility is confirmed.

--- origsrc/libmodplug-0.8.8.5/src/modplug.h	2014-03-09 18:49:07.000000000 -0500
+++ src/libmodplug-0.8.8.5/src/modplug.h	2015-06-12 16:55:49.000558600 -0500
@@ -11,7 +11,7 @@
 extern "C" {
 #endif
 
-#if defined(_WIN32) || defined(__CYGWIN__)
+#if defined(_WIN32)
 # if defined(MODPLUG_BUILD) && defined(DLL_EXPORT)	/* building libmodplug as a dll for windows */
 #   define MODPLUG_EXPORT __declspec(dllexport)
 # elif defined(MODPLUG_BUILD) || defined(MODPLUG_STATIC)	/* building or using static libmodplug for windows */
