--- origsrc/libmodplug-0.8.4/libmodplug.pc.in	2005-07-04 19:20:34.000000000 -0500
+++ src/libmodplug-0.8.4/libmodplug.pc.in	2007-11-09 02:40:21.953125000 -0600
@@ -7,5 +7,5 @@
 Description: The ModPlug mod file playing library.
 Version: @VERSION@
 Requires: 
-Libs: -L${libdir} -lmodplug -lstdc++ -lm
+Libs: -L${libdir} -lmodplug -lm
 Cflags: -I${includedir} -I${includedir}/libmodplug
--- origsrc/libmodplug-0.8.4/src/Makefile.am	2006-11-01 21:48:13.000000000 -0600
+++ src/libmodplug-0.8.4/src/Makefile.am	2007-11-09 02:39:20.562500000 -0600
@@ -1,5 +1,6 @@
-INCLUDES = -Ilibmodplug
+INCLUDES = -I$(srcdir)/libmodplug
 lib_LTLIBRARIES = libmodplug.la
+libmodplug_la_LDFLAGS = -no-undefined
 libmodplug_la_LIBADD = -lm
 libmodplug_la_SOURCES = tables.cpp         \
                         sndmix.cpp         \
